Content.Client/Light/Visualizers/ExpendableLightVisualizerComponent.cs

```diff
@@ -0,0 +1,12 @@
+namespace Content.Client.Light.Visualizers;
+
+[RegisterComponent]
+[Access(typeof(ExpendableLightVisualizerSystem))]
+public sealed class ExpendableLightVisualizerComponent : Component
+{
+    [DataField("iconStateSpent")]
+    public string? IconStateSpent { get; set; }
+
+    [DataField("iconStateOn")]
+    public string? IconStateLit { get; set; }
```

Comment:
i dont think the getter/setter is necessary, also make these mapped layers or SpriteSpecifier